src/Sarif/Core/PropertyBagHolder.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.CodeAnalysis.Sarif
     /// <summary>
     /// Base class for objects that can hold properties of arbitrary types.
     /// </summary>
-    public abstract class PropertyBagHolder : IPropertyBagHolder
+    public class PropertyBagHolder : IPropertyBagHolder
```

Comment:
>class [](start = 11, length = 5)

not strictly a required change. i am observing odd cases where serialization isn't eliding default values, thought this might help. i don't see a need to retain this class as abstract, though, so left this in.

Reply:
Sure, fine.

---
In reply to: [249128719](https://github.com/Microsoft/sarif-sdk/pull/1223#discussion_r249128719) [](ancestors = 249128719)